flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/config/Environment.java

```diff
@@ -48,6 +49,8 @@
 
 	public static final String EXECUTION_ENTRY = "execution";
 
+	public static final String CONFIGURATION_ENTRY = "table";
```

Comment:
What about set CONFIGURATION_ENTRY to 'configuration' (Just like execution or deployment segments)
And user use "SET configuration.table.exec.x.y.z=a" to set "table.exec.x.y.z".

Reply:
And the ConfigurationEntry should provide an 'asTopLevelMap' interface for accessing entities with 'configuration' prefix

Reply:
Hi @docete ,  we would like to avoid add a `configuration` prefix when SET config options, so that users can have a similar experience with `TableConfig`. The `CONFIGURATION_ENTRY` here is used to match configuration key prefix (all the existing configuration key are started with `table.exec` or `table.optimizer`)

Reply:
I agree with Jark. We should not have another level of nesting when using the `SET` command.